corehq/apps/hqadmin/views.py

```diff
@@ -472,6 +472,12 @@ def get(self, request, *args, **kwargs):
             response, _ = self._get_restore_response()
             return response
 
+        download = request.GET.get('download') == 'true'
+        if download:
+            response, _ = self._get_restore_response()
+            response['Content-Disposition'] = "attachment; filename={}-restore.xml".format(username)
```

Comment:
is there possibly a unicode issue here? (forget if we ever allow unicode usernames, though thought we might have at one point)